doc/2-interface/menu-bar.md

```diff
@@ -2,24 +2,20 @@
 
 the menu bar allows you to select from five menus: file, edit, settings, window and help.
 
-items in _italic_ don't appear in basic mode and are only available in advanced mode.
-
 ## file
 
-- **new...**: creates a new song.
+- **new...**: opens the new song dialog to choose a system.
+  - click a system name to create a new song with it.
+  - some systems have several variants and are grouped, indicated by with a triangle to the left of the name. the first click on a group opens it. clicking again on the group name will create a new song with it.
```

Comment:
I think this could be improved.

Maybe just "some systems have several variants, which are inside a group."